src/components/pages/SingleEventPage.tsx

```diff
@@ -89,10 +91,84 @@ class SingleEventPage extends React.Component<Props, State> {
           <div className={ classes.event }>
             { this.renderEventPicture() }
             <div className={ classes.eventColumn } id="readthis">
-              { fetchedContent.offers[0].price.fi &&
-                <Typography variant="h6" style={{ marginBottom: "1rem" }}> { fetchedContent.offers[0].price.fi }</Typography>
-              }
-              <Typography variant="body2">{ fetchedContent.description.fi }</Typography>
+              <Card className={ classes.card }>
+                <CardContent>
+                  <Typography variant="h3" className={ classes.heroText }>
+                    { fetchedContent ? fetchedContent.name.fi || "Event" : "Event" }
+                  </Typography>
+                  <Typography variant="body2">
+                    { fetchedContent.description.fi }
+                  </Typography>
+                  <Typography variant="body2">
+                    { fetchedContent.location.id }
+                  </Typography>
+                </CardContent>
+              </Card>
+              <Card className={ classes.card }>
+                <CardContent>
+                  <Typography variant="h6">
+                    Tapahtuman tiedot
+                  </Typography>
+                  { moment(fetchedContent.start_time).format("DD.MM.YYYY HH:mm") &&
+                    <Typography variant="body2">
+                      Alkaa: { moment(fetchedContent.start_time).format("DD.MM.YYYY HH:mm") }
+                    </Typography>
+                  }
+                  { moment(fetchedContent.end_time).format("DD.MM.YYYY HH:mm") &&
+                    <Typography variant="body2">
+                      Päättyy: { moment(fetchedContent.end_time).format("DD.MM.YYYY HH:mm") }
+                    </Typography>
+                  }
```

Comment:
Is there a reason to check that `moment(fetchedContent.start_time).format("DD.MM.YYYY HH:mm")` exists before rendering? If there is, move it first in its own variable to improve readability. For example `const startTime = moment(fetchedContent.start_time).format("DD.MM.YYYY HH:mm")`

Reply:
same for end time